docs/python_virtual_environments.rst

```diff
@@ -40,6 +40,11 @@ To activate an environment, run ``vox activate <envname>``::
 
 Instead of ``activate``, you can call ``workon`` or ``enter``.
 
+If you want to activate an environment which is stored somewhere else (maybe because it was created by another tool) you can pass to ``vox deactivate`` a path to a virtual environment::
```

Comment:
Typo.

Reply:
Thanks @astronouth7303. What *is* the typo? And does it resolve the failing of the CI checks? I am a bit at a loss with the CI errors...

Reply:
Thanks @astronouth7303. Fixed it in this branch.